plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzers/events_log_analyzer.py

```diff
@@ -152,3 +147,11 @@ def plot_link_up_down_count_per_aggregation_time(self):
         self._save_pivot_data_in_bars(
             pivot_links_data, "Time", "Number of Events", "Link up/down events", "Event"
         )
+
+    def full_analysis(self):
+        super().full_analysis()
+        df_to_add = (
+            "More than 5 events burst over a minute",
+            self.get_critical_event_bursts(),
```

Comment:
Why does it need to be here, why not like the images? where each analyzing function is run autmaticly and sets the output....

For example. in this case, get_critical_event_bursts once called, will add the data to the `dataframes_for_pdf`

Reply:
fixed